ChemistryLib/PhreeqcInterfaceData/EquilibriumPhase.h

```diff
@@ -0,0 +1,45 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <boost/optional/optional.hpp>
+#include <fstream>
+#include <string>
+#include <vector>
+
+namespace BaseLib
+{
+class ConfigTree;
+}
+
+namespace ChemistryLib
+{
+class EquilibriumPhase
+{
+public:
+    EquilibriumPhase(std::string name, double amount, double saturation_index)
+        : _name(name), _amount(amount), _saturation_index(saturation_index)
+    {
+    }
+
+    friend std::ofstream& operator<<(
+        std::ofstream& outFile,
+        std::vector<EquilibriumPhase> const& equilibrium_phases);
+
+    std::string const _name;
+    double _amount;
```

Comment:
public members don't have underscores...

Reply:
Fixed also in struct KineticReactant and struct Component.

Reply:
Same in the `Solution` class...

Reply:
Fixed.